File: 242-valid-anagram/valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if len(s)!=len(t):
            return False
        freq=[0]*26
        for c in s:
            freq[ord(c)-ord('a')]+=1
        for c in t:
            freq[ord(c)-ord('a')]-=1
        for count in freq:
            if count!=0:
                return False
        return True

File: 242-valid-anagram/valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length()!=t.length())
        return false;
        int[] freq=new int[26];
        for(char c:s.toCharArray()){
            freq[c-'a']++;
        }
        for(char c:t.toCharArray()){
            freq[c-'a']--;
        }
        for(int count:freq){
        if(count!=0)return false;}
        return true;
    }
}